docs/content/en/docs/getting-started/optional/api-server-extra-args.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "API Server Extra Args"
+linkTitle: "API Server Extra Args"
+weight: 10
```

Comment:
The weight sets where in the left nave this page will appear, in relation to other pages in the "optional" directory. This will cause it to be listed somewhere between the second and fourth entry. If you would prefer this to show up further down the list, make the number higher. I have no preference.